spec/features/admin/pages_spec.rb

```diff
@@ -1,7 +1,13 @@
 require 'rails_helper'
 
 describe 'admin pages' do
-  let!(:account){ FactoryGirl.create :confirmed_account }
+  let!(:account) do
+    FactoryGirl.create(:confirmed_account).tap do |account|
+      Theblog::Role.find_by(name: :editor).accounts << account
+    end
+  end
+  let(:another_account) { FactoryGirl.create :confirmed_account }
```

Comment:
actually 'let' is not required here, you might move them into
https://github.com/kont-noor/Theblog/pull/16/files#diff-c8c4011dcb972f71076e4f2b05e8029eR4 like
```ruby
let!(:account) do 
another_acc = FactoryGirl.create :confirmed_account
FactoryGirl.create :page, author: another_acc
FactoryGirl.create(:editor_account) 
end
```
or even via 'before' 
```ruby
before do
another_acc = FactoryGirl.create :confirmed_account
FactoryGirl.create :page, author: another_acc
@account = FactoryGirl.create(:editor_account) 
end
```